docs/guides/flux.md

```diff
@@ -110,35 +110,69 @@ So we now have an action called `userValueUpdated`, which when called dispatches
 ### Connecting a React Component to our Flux Store
 
 The store is now updating its data - but we have no React components connected to the store! Lets set one up:
+Here is an example of how to set up a connected Component. It shows several ways that a component can connect to Stores or simply be overridden with props.
 
 ```js
 // ./src/views/user/index.js
 
 import React from 'react';
 import { connect } from 'utils/flux';
-import Textbox from 'carbon-react/lib/components/textbox';
+import AccountInvitation from 'test/lib/components/accountinvitation';
 import UserStore from 'stores/user';
+import GiftsStore from 'stores/gifts';
 import UserActions from 'actions/user';
 
-class UserView extends React.Component {
+class AccountInvitation extends React.Component {
   render() {
     return (
-      <Textbox name="foobar"
-        value={ this.state.userStore.get('foobar') }
-        onChange={ UserActions.userValueUpdated } />
+      <div>
+        <Textbox name="invitationidTextBox"
+          value={ props.invitationid }
+        />
+        <Textbox name="firstNameTextBox"
+          value={ props.firstname }
+          onChange={ UserActions.userFirstNameValueUpdated } />
+        />
+        <Textbox name="giftTextBox"
+          value={ props.gift }
+        />
+      </div>
     );
   }
 }
+const mapStateToProps = (userState, giftsState, props) => {
+  return {
+    invitationid: props.invitationid,
+    firstname: props.firstname || userState.get('firstname'),
```

Comment:
I'm not sure you want this in an example. I think it might be a bit of a bad pattern as this would be pushing the responsibility of setting a sensible default to the component developer and not the consumer. If it did need a value it would probably be marked as a required prop.